app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentSubscriptionConnector.scala

```diff
@@ -135,15 +137,14 @@ class AgentSubscriptionConnector @Inject()(
       }
     }
 
-  def matchCitizenDetails(citizenDetailsRequest: CitizenDetailsRequest)(implicit hc: HeaderCarrier): Future[Boolean] =
-    monitor(s"ConsumedAPI-Agent-Subscription-getCitizenDetails-POST") {
-      val endpoint = s"/agent-subscription/citizen-details"
+  def getDesignatoryDetails(nino: Nino)(implicit hc: HeaderCarrier): Future[DesignatoryDetails] =
+    monitor(s"ConsumedAPI-Agent-Subscription-getDesignatoryDetails-GET") {
+      val url = new URL(baseUrl, s"/agent-subscription/citizen-details/${nino.value}/designatory-details").toString
       http
-        .POST[CitizenDetailsRequest, HttpResponse](new URL(baseUrl, endpoint).toString, citizenDetailsRequest)
-        .map(_.status == 200)
+        .GET[DesignatoryDetails](url)
         .recover {
-          case e: BadRequestException => false
-          case f: NotFoundException   => false
+          case f: NotFoundException =>
```

Comment:
I think we should at least have a Logger.warn() or Logger.error() here?  How expected is this?

Reply:
I added a warning log in the postcode controller

Reply:
thank you!